sdk/enter.go

```diff
@@ -365,11 +363,11 @@ func enterChrootHelper(args []string) (err error) {
 		return err
 	}
 
-	if err := e.MountAgent("SSH_AUTH_SOCK"); err != nil {
+	if err := e.mountAgent("SSH_AUTH_SOCK"); err != nil {
```

Comment:
It seems weird to pass in an env var name here, but to assume GNUPG home in mountGnupg a little below.

I'd rather just not pass it in and let the function assume for consistency; it's not really buying us anything to abstract it.